website/src/components/Survey/SurveyCard.tsx

```diff
@@ -1,22 +1,18 @@
 import { Box, BoxProps, useColorModeValue } from "@chakra-ui/react";
+import { PropsWithChildren } from "react";
 
-interface SurveyCardProps {
-  className?: string;
-  children: React.ReactNode;
-}
-
-export const SurveyCard = (props: SurveyCardProps) => {
+export const SurveyCard = (props: PropsWithChildren<{ className?: string }>) => {
   const backgroundColor = useColorModeValue("white", "gray.700");
 
   const BoxClasses: BoxProps = {
     gap: "2",
     borderRadius: "xl",
     shadow: "base",
-    className: "p-4 sm:p-6",
+    className: "p-4 sm:p-6 " + (props.className ?? ""),
```

Comment:
You can use `clsx` here to simplify the code

Reply:
Of cousre!